### DISCUSSION TITLE
Will adjustable parameter for 0 wt obs (only) affect overall solution in IES?

### DISCUSSION THREAD

**Discussion by wkitlasten** [usgs/pestpp#209] [General]
I have a group of parameters that are associated with a single observation.

In one set of simulations the obs weight is non-zero, in the other set the obs weight is 0. Unfortunately I did not set partrans to "fixed" in the control file for those parameters associated with 0 weight obs, but my localizer is (absurdly?) granular with 0 for all obs except the single obs associated with that group of parameters.

The parameter change summary shows very small mean change in the parameters that should have been fixed compared to the smallest change in other adjustable parameters (1e-13, approximately <1e-8 times smaller than other parameters). 

I'm assuming the very small change is due do to numerical precision in the parameter upgrade solution and that it does not affect the overall solution. Is that a fair assumption? Are there potential complications that warrant rerunning the simulations (12+ hrs on 150 cores)?

I am using IES with 300 realizations.

**Comment by jtwhite79** [comment]
1e-13 sounds like zero to me! haha

You could load up the initial and final par ensembles and invest these changes if you are uncomfortable with the par change summary...

**Comment by wkitlasten** [comment]
Ha! Yes, 1e-13 par change is small enough for me and the par dist for those are essentially identical.

I was more worried about some other aspect I might be overlooking (since I'm not so good with maths and computers and stuff)... like horribly insensitive adjustable parameters corrupting the upgrade in some way.

**Comment by jtwhite79** [comment]
I see.  I think the truncated SVD solve should lock out horribly insensitive parameter (component)s from adjustment - maths!
